Makefile

```diff
@@ -4,6 +4,8 @@ VERSION ?= SNAPSHOT-${GIT_COMMIT_SHORT}
 LDFLAGS_VERSION=-X github.com/astronomer/astro-cli/version.CurrVersion=${VERSION}
 
 CORE_OPENAPI_SPEC=../astro/apps/core/docs/public/v1alpha1/public_v1alpha1.yaml
+CORE_IAM_OPENAPI_SPEC=../astro/apps/core/docs/iam/v1beta1/iam_v1beta1.yaml
+CORE_PLATFORM_OPENAPI_SPEC=../astro/apps/core/docs/platform/v1beta1/platform_v1beta1.yaml
```

Comment:
i think we might wanna add v1beta1 to this in case we add other versions later so we don't get confused